blocks/form/form.js

```diff
@@ -135,6 +135,10 @@ function createInput(fd) {
   input.id = fd.Field;
   input.name = fd.Field;
   input.setAttribute('placeholder', fd.Placeholder);
+  input.setAttribute('maxlength', fd.Options);
```

Comment:
I would suggest using the maxLength/maximum columns instead of Options column. Options will confuse the authors.